roles/dosomething.jenkins/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: Add Jenkins Repo Key
+  become: yes
+  apt_key: url=https://jenkins-ci.org/debian/jenkins-ci.org.key
+
+- name: Add Jenkins Repo
+  become: yes
+  apt_repository: repo='deb http://pkg.jenkins-ci.org/debian binary/' state=present update_cache=yes
```

Comment:
I know it is, but I like to enumerate in case they ever default to "no" or syntax changes. This isn't really a bad practice, and I've seen it in Ansible examples before.